devai-cli/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
I think we need to bring `requirements-test.txt` back to install `pytest` so that we can run the tests.

Reply:
did you run 
```
python3 -m venv venv
. venv/bin/activate
pip install -r src/requirements.txt
```
that should have loaded the requirements.

Reply:
I did.

command not found: pytest